content/kotlin/concepts/datetime/terms/periodUntil/periodUntil.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: '.periodUntil()'
+Description: 'returns an object representing the difference between two dates in terms of years, months and days.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **periodUntil()** function is used to calculate the difference between two `LocalDate` or `Instant` instances. This returns an `object` representing the difference between the two dates in terms of years, months and days.
+
+## Syntax
+
+```pseudo
+expect fun LocalDate.periodUntil(other: LocalDate): DatePeriod
+```
+
+`DatePeriod`gives the difference between two instants, decomposed into date and time components where all time components are equals to zero.
```

Comment:
Use bullet points to explain the parameters and explain the LocalDate parameter too.